README.md

```diff
@@ -19,10 +23,10 @@ cd imas-data-dictionary
 pip install . [--user]
```

Comment:
I think these three lines could be shorted to something like:
`pip install git+https://github.com/iterorganization/IMAS-Data-Dictionary.git [--user]`

Reply:
Okay.